backend/api/manageUsersApi.js

```diff
@@ -80,6 +81,7 @@ const manageUsersApiFactory = (client) => {
     userGroups,
     addUserGroup,
     removeUserGroup,
+    enableUser,
```

Comment:
I think we're gonna have to start to distinguish between nomis and external users
I have started to use externalUser for the new types (not sure if that's the best name?) - as we will be moving the enableUser (from nomisUserRolesApi.js) at some point soon

Reply:
updated to enableExternalUser